db/migrate/20191029163132_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    enable_extension "pgcrypto" unless extension_enabled?("pgcrypto")
+
+    create_table :users, id: :uuid do |t|
+      t.string :identifier, index: true
+      t.string :name
+      t.string :email
```

Comment:
Whilst we are quite sure email address will be consumed later to facilitate user creation, I don't think it strictly needs to be in this pull request. 

Storing names and email addresses is the start of us handling personally identifiable information so worth checking the needs in an effort to document them. I think we need both because:

- We need the users name to present back to them confirming they are recognised and increasing trust in the service
- We need the users name to store in our audit trail to identify people who have performed actions within the service. This increases accountability and aids us in recovering from the actions of a compromised set of credentials
- We need the users email address to that a BEIS admin can confirm they have invited the correct person